src/main/java/com/swervedrivespecialties/exampleswerve/util/BeakXboxController.java

```diff
@@ -1,13 +1,50 @@
 package com.swervedrivespecialties.autonrefresh.util;
 
-import com.swervedrivespecialties.autonrefresh.RobotMap;
-
 import edu.wpi.first.wpilibj.XboxController;
 import edu.wpi.first.wpilibj2.command.button.JoystickButton;
+import edu.wpi.first.wpilibj.GenericHID.Hand;
+import edu.wpi.first.wpilibj2.command.button.Button;
 
 public class BeakXboxController{
 
-    private XboxController controller;
+    //these constants all come from the layout of our physical controllers
+    //and their IO interfaces with WPILib
+
+    //deadband
+    private static final double DEADBAND = .05;
+
+    //button indexing
+    private static final int BUTTON_A_INDEX = 1;
+    private static final int BUTTON_B_INDEX = 2;
+    private static final int BUTTON_X_INDEX = 3;
+    private static final int BUTTON_Y_INDEX = 4;
+    private static final int BUTTON_LB_INDEX = 5;
+    private static final int BUTTON_RB_INDEX = 6;
+    private static final int BUTTON_BACK_INDEX = 7;
+    private static final int BUTTON_START_INDEX = 8;
+
+    //trigger identification booleans
+    private static final boolean IS_LEFT_TRIGGER_DEFAULT = true;
+    private static final boolean LEFT_TRIGGER_BOOLEAN = IS_LEFT_TRIGGER_DEFAULT;
+    private static final boolean RIGHT_TRIGGER_BOOLEAN = !IS_LEFT_TRIGGER_DEFAULT;
+
+    //axis indexing
+    private static final int LEFT_X_AXIS_INDEX = 0;
```

Comment:
I tried to add all of these. You made changes to this class, which is a very good thing, and it seemed like this was the style you were going for, so I went with everything as constants